### PR TITLE
Add checking for localStorage token's props on Cypress tests

### PR DESCRIPTION
**Pull request by JustCaptcha**

### CODE REVIEW

File: universal-login-wallet/cypress/support/loginAppCommands.ts [hunk 1]
```diff
@@ -10,7 +10,9 @@ Cypress.Commands.add('pickUsername', (name) => {
   cy.get('.unilogin-component-suggestions-list').first()
     .get('.unilogin-component-suggestions-ens-name')
     .get('p.unilogin-component-ens-name').contains(name);
-  cy.get('.unilogin-component-suggestions-list').first().find('#create-new').click();
+  cy.get('.unilogin-component-suggestions-list').first().find('#create-new').click().should(() => {
+    expect(localStorage.getItem('wallet-ganache.kind')).to.eq = 'Future4';
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/cypress/support/loginAppCommands.ts:14]: Do we need this `4`? 

Also, I don't think that we should put here `expect`. This file contains the setup for tests and shouldn't be a test itself.

File: universal-login-wallet/cypress/support/loginAppCommands.ts [hunk 2]
```diff
@@ -28,3 +28,11 @@ Cypress.Commands.add('topUpAccount', () => {
...
   cy.get('.modal-success-btn').click();
 });
+
+Cypress.Commands.add('getLocalStorage', (token) => {
+  cy.window().then((window) => JSON.parse(window.localStorage.getItem(token)));
+});
+
+Cypress.Commands.add('checkLocalStorageTokenProp', (token, prop, value) => {
+  cy.getLocalStorage(token).its('kind').should('eq', value);
+});
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/cypress/support/loginAppCommands.ts:38]: Could we please move it to `commands.js` file?

Comment (JustCaptcha) [reply universal-login-wallet/cypress/support/loginAppCommands.ts:38]: Ok.

File: universal-login-wallet/cypress/integration/createAccount.test.ts [hunk 3]
```diff
@@ -9,11 +9,11 @@ describe('E2E: Create Account', () => {
       cy.approveTerms();
       cy.ensureCorrectLocation('/selectDeployName');
       cy.pickUsername(ensName);
-      cy.ensureCorrectLocation('/create/topUp');
-      cy.selectTopUpWithCrypto();
-      cy.topUpAccount();
+      cy.wait(400);
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/cypress/integration/createAccount.test.ts:12]: We should use something different. Something like waitExpect

**Pull request merged**